meal_planner/database.py

```
@@ -2,13 +1,49 @@
+"""Database connection and session management for the Meal Planner application.
 
-from sqlmodel import Session, create_engine
+This module provides SQLModel/SQLAlchemy database engine configuration and
+session management utilities. It handles the creation of database connections
+and provides functions for initializing the database schema.
+"""
+
+from sqlmodel import Session, SQLModel, create_engine, select
```

Ruff (0.11.9)

`sqlmodel.select` imported but unused

Remove unused import: `sqlmodel.select`

(F401)